VideoLocker/res/layout/drawer_navigation.xml

```diff
@@ -11,34 +12,51 @@
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
         android:background="@color/grey_name_background"
-        android:orientation="vertical"
-        android:padding="15dp">
+        android:orientation="horizontal"
+        android:padding="@dimen/edx_margin"
+        android:gravity="center">
+
+        <de.hdodenhof.circleimageview.CircleImageView
+            android:id="@+id/profile_image"
+            android:layout_width="70dp"
+            android:layout_height="70dp"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_loading_placeholder"
+            android:layout_marginRight="@dimen/edx_margin"
```

Comment:
There should be a corresponding `layout_marginEnd` too, to support RTL mode on supported devices.